test/sys/test_socket.rs

```diff
@@ -683,3 +683,66 @@ pub fn test_recv_ipv6pktinfo() {
 
     thread.join().unwrap();
 }
+
+#[cfg(any(
+target_os = "linux",
+))]
+#[test]
+pub fn test_mmsg() {
+    use std::thread;
+    use nix::sys::uio::IoVec;
+    use nix::sys::socket::{socket, bind, connect, recvmmsg, sendmmsg, MMsgHdr, MsgFlags, AddressFamily, SockType, SockAddr, SockFlag, InetAddr};
+    use std::time;
+
+    let sender = thread::spawn(move || {
```

Comment:
You probably don't need to spawn a thread here.  You should try removing it, and the test will probably still work.  The order of operations should be:
1) Create and bind the receiving socket
2) Created, bind, and connect the sending socket
3) sendmmsg
4) recvmmsg